ts/components/conversation/message/message-content/MessageContent.tsx

```diff
@@ -88,11 +43,19 @@ function onClickOnMessageInnerContainer(event: React.MouseEvent<HTMLDivElement>)
   }
 }
 
-const StyledMessageContent = styled.div`
-  border-radius: 18px;
+const StyledMessageOpaqueContent = styled.div<{ messageDirection: MessageModelType }>`
+  background: ${props =>
+    props.messageDirection === 'incoming'
+      ? 'var(--color-received-message-background)'
+      : 'var(--color-sent-message-background)'};
+  align-self: ${props => (props.messageDirection === 'incoming' ? 'flex-start' : 'flex-end')};
+
+  padding: 7px 13px; // FIXME
```

Comment:
What needs to be fixed? Or is this for theming?

Reply:
yes!